CONTRIBUTING.md

```diff
@@ -0,0 +1,19 @@
+We love pull requests from everyone. Follow the thoughtbot [code of conduct]
+while contributing.
+
+[code of conduct]: https://thoughtbot.com/open-source-code-of-conduct
+
+## Contributing
+
+1. Fork the repo.
+2. Push to your fork and submit a pull request.
+
+At this point you're waiting on us. We like to at least comment on, if not
+accept, pull requests within three business days (and, typically, one business
+day). We may suggest some changes or improvements or alternatives.
+
+Some things that will increase the chance that your pull request is accepted,
+
+* Update the documentation, the surrounding one, examples elsewhere, guides,
```

Comment:
This sentence reads strangely, I'm not quite sure what "the surrounding one" means?